containers/allan_dip_bot/run_bot.py

```diff
@@ -74,6 +76,7 @@ async def launch(
     sleep_delay: bool,
     discount_factor: float,
     outdir: Optional[Path],
+    aggressiveness: Aggressiveness = Aggressiveness.moderate,
```

Comment:
Can you add `aggressiveness` after `discount_factor` in all places in this file? They both are used to instantiate the same type of bot, so it makes sense to group them together.

Reply:
The ordering you suggest was what the order previously was. However, I was getting the error "Non-default argument follows default argument" which was solved by moving the aggressiveness parameter to the end.

Reply:
Yes, you're right! It has to go there. I forgot about that.